### PR TITLE
D2IQ-67223: add 'os' attributes for Linux agents by default

### PR DESCRIPTION
**Pull request by sergiimatusEPAM**
## High-level description

Having Windows agent introduced with "os" attributes where value is "windows", the Linux agent needs to be distinguished. Therefore adding "os:linux" for all Linux slaves by default, regardless it's private or public.

## Corresponding DC/OS tickets (required)

  - [D2IQ-67223](https://jira.d2iq.com/browse/D2IQ-67223) Linux nodes are not flagged with the os:linux attribute

## Related tickets (optional)

  - [D2IQ-ID](https://jira.mesosphere.com/browse/D2IQ-<number>) JIRA title / short description.

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by sergiimatusEPAM** [comment]
@mesosphere-mergebot request-review @vinodkone

**Comment by sergiimatusEPAM** [comment]
@mesosphere-mergebot request-review @DominikDary

**Comment by sergiimatusEPAM** [comment]
@mesosphere-mergebot request-review @jongiddy

**Comment by sergiimatusEPAM** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by sergiimatusEPAM** [comment]
@mesosphere-mergebot changelog-not-required Windows beta4 minor configuration change

**Comment by vinodkone** [comment]
> @mesosphere-mergebot changelog-not-required Windows beta4 minor configuration change

@sergiimatusEPAM  Can we actually note this attribute addition in the CHANGELOG? I think it's worth calling out?

File: CHANGES.md [hunk 1]
```diff
@@ -95,6 +95,7 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
 
 * Adjust dcos-net (l4lb) to allow for graceful shutdown of connections by changing the VIP backend weight to `0`
   when tasks are unhealthy or enter the `TASK_KILLING` state instead of removing them. (D2IQ-61077)
+* Set "os:linux" attribute for the Linux agents. (D2IQ-67223)
```
Comment (cneth) [inline CHANGES.md:98]: Some DC/OS customers already set and use the 'os' attribute.   I suggested we namespace this as 'dcos.os' or something similar.

Comment (orsenthil) [reply CHANGES.md:98]: @cneth - the PR was approved and had been merged in a train.  If you think, it is a critical issue, please raise a ticket.

**Pull request merged**